language/language_quickstart/main.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START language_quickstart]
+// Sample language_quickstart uses the Google Cloud Natural API to analyze the
+// sentiment of "Hello, world!".
+package main
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
```

Comment:
Move the import to the next group.

The top group belongs to the standard library imports.